.github/workflows/ci.yml

```
@@ -22,7 +22,7 @@ jobs:
         with:
           dotnet-version: 5.0.x
       - name: Add msbuild to PATH
-        uses: microsoft/setup-msbuild@v1.0.2
+        uses: microsoft/setup-msbuild@v1.0.3
```

I don't know why they don't use 3-number format but 1.0.3. still uses old node.
```suggestion
        uses: microsoft/setup-msbuild@v1.3
```